src/util-profiling.c

```diff
@@ -1309,6 +1309,7 @@ const char * PacketProfileLoggertIdToString(LoggerId id)
         CASE_CODE (LOGGER_JSON_HTTP);
         CASE_CODE (LOGGER_JSON_DHCP);
         CASE_CODE (LOGGER_JSON_KRB5);
+        CASE_CODE(LOGGER_JSON_MODBUS);
```

Comment:
Did clang-format not like a space after `CASE_CODE` ?

Reply:
Yes, this was `clang-format`'s doing.